### ISSUE TITLE
cannot reload 

### ISSUE THREAD

**Issue by alexbijma** [nextcloud/deck#2890]
my Deck Android app was working correct. but today it didn't refresh, I made some changes on the web. As i didn't got the changes on my Android app I tried to refresh multiple times. Couple of times hardclosed the app and restarted and then the app crashed and I got the below report.

```
Full Crash:

App Version: 1.15.4
App Version Code: 1015004
App Flavor: play

Files App Version Code: 30150190

---

OS Version: 4.14.117-20147032(A705FNXXU5CUA7)
OS API Level: 29
Device: a70q
Manufacturer: samsung
Model (and Product): SM-A705FN (a70qxx)

---

java.lang.NullPointerException: Attempt to invoke interface method 'io.reactivex.Observable it.niedermann.nextcloud.deck.api.NextcloudServerAPI.getCapabilities(java.lang.String)' on a null object reference
at it.niedermann.nextcloud.deck.persistence.sync.adapters.ServerAdapter.lambda$getCapabilities$1$ServerAdapter(ServerAdapter.java:138)
at it.niedermann.nextcloud.deck.persistence.sync.adapters.-$$Lambda$ServerAdapter$loCQuSjClPIgc95_PUtPXCMl42k.getObservableFromCall(Unknown Source:4)
at it.niedermann.nextcloud.deck.api.RequestHelper.request(RequestHelper.java:31)
at it.niedermann.nextcloud.deck.persistence.sync.adapters.ServerAdapter.getCapabilities(ServerAdapter.java:138)
at it.niedermann.nextcloud.deck.persistence.sync.SyncManager.lambda$refreshCapabilities$6$SyncManager(SyncManager.java:436)
at it.niedermann.nextcloud.deck.persistence.sync.-$$Lambda$SyncManager$vxp4ju9aFijCoAe_PhyyGT1M1Ig.run(Unknown Source:4)
at java.lang.Thread.run(Thread.java:919)

**Comment by stefan-niedermann** [comment]
Dear @alexbijma

this repository is *only* for the Web UI / server part of Deck. For the Android client you can open bug reports with a completely filled out issue template [here](https://github.com/stefan-niedermann/nextcloud-deck/issues/new?assignees=&labels=bug&template=bug_report.md)

Before open a new bug report please make sure to read our [FAQs](https://github.com/stefan-niedermann/nextcloud-deck/blob/master/FAQ.md#i-have-experienced-an-error) first and try to clear the storage as described, this might already end the weird state of your app. Also make sure to update to the latest available version - 1.15.5 fixed a bug which can cause the state you described if the deck server version is lower than 1.3.0. The new version is already published at Play Store but unfortunately delayed by Google because of Corona.

Sorry for the inconveniences and kind regards,

Stefan

**Comment by alexbijma** [comment]
the update fixed the issue indeed. thanks!

sorry for the inconvenience by posting it in this repository.
